zulipterminal/model.py

```diff
@@ -492,6 +492,28 @@ def append_message(self, event: Event) -> None:
                 last_message = self.msg_list.log[-1].original_widget.message
             else:
                 last_message = None
+            # Removing the dummy message when a new message comes
+            dummy_message_found = True
+            if self.msg_list.log:
+                possible_dummy_message = \
+                    self.msg_list.log[0].original_widget.message
+                if possible_dummy_message.get('id') is None:
```

Comment:
Did you choose `.get` here intentionally, over indexing? We're checking for the special value for when it's the dummy case, right?